src/styles/design/_infinite-scroll.scss

```diff
@@ -0,0 +1,14 @@
+.item-scroll {
+    @apply cursor-pointer p-2 border-b border-gray-300; 
+    &:hover {
+      @apply bg-blue-100;
+    }
+}
+
+.item-selected {
```

Comment:
Can we constrain these classes to the infinite scroll container since the names are generic?